app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -1,20 +1,20 @@
 # frozen_string_literal: true
 
 class OmniauthCallbacksController < Devise::OmniauthCallbacksController
-  # rubocop:disable MethodLength
   def google_oauth2
-    @user = User.find_for_google_oauth2(
-      request.env['omniauth.auth'],
-      current_user
-    )
-
-    if @user
+    if user
```

Comment:
Can we use `user.present?` or something like that to make it a bit more explicit what this is doing